src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1068,16 +1068,20 @@ public void checkout(Run<?, ?> build, Launcher launcher, FilePath workspace, Tas
 
         try {
           checkoutCommand.execute();
-        } catch(GitLockFailedException e) {
+        } catch (GitLockFailedException e) {
```

Comment:
Nah, changed my mind, not unfixing it, because it'll get unfixed by some future commit by someone else anyway!